plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenArchetypeProjectWizard.java

```diff
@@ -42,10 +45,14 @@ public MavenArchetypeProjectWizard() {
 
   @Override
   public void addPages() {
-    page = new MavenAppEngineStandardWizardPage();
-    archetypePage = new MavenAppEngineStandardArchetypeWizardPage();
-    this.addPage(page);
-    this.addPage(archetypePage);
+    if (doesAppEngineJavaComponentExist()) {
+      page = new MavenAppEngineStandardWizardPage();
+      archetypePage = new MavenAppEngineStandardArchetypeWizardPage();
+      this.addPage(page);
+      this.addPage(archetypePage);
+    } else {
+      this.addPage(new AppEngineComponentPage());
```

Comment:
There is a problem with Analytics. `AddEngineComponentPage()` sends a ping with `type=native`. Should send `type=maven`.